src/utils.js

```diff
@@ -1413,26 +1412,6 @@ function syntaxErrorFactory(error) {
   return obj;
 }
 
-async function writeModulesMap(getJSON, resourcePath, exports) {
```

Comment:
I removed this function since the logic now is pretty simple in [src/index.js](https://github.com/webpack-contrib/css-loader/pull/1577/commits/0009dd9a6036252b25352c82e0210b36e5ce86c6#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R277-R289).  Happy to keep it as a simple function in src/utils.js if preferred, though!